app/controllers/application_controller.rb

```diff
@@ -396,6 +396,10 @@ def render_not_found
     render template: 'pages/page_not_found', layout: false, status: :not_found, formats: :html
   end
 
+  def render_json_not_found
+    render json: { error: "The page you were looking for doesn't exist" }, status: :not_found
+  end
```

Comment:
What if we combined this with `render_not_found` into:
```suggestion
    respond_to do |format|
      format.html { render template: 'pages/page_not_found', layout: false, status: :not_found, formats: :html }
      format.json { render json: { error: "The page you were looking for doesn't exist" }, status: :not_found } 
    end
  end
```

Reply:
Ah, yeah, I think that makes more sense. I think I was quite fixated on just adapting to use the existing method (this was also before I realized that `render_api_not_found` wasn't already an `ApplicationController` method).

I'll update.

Reply:
Updated in 5b91848 + 2b05423.